elasticdl_client/common/k8s_client.py

```diff
@@ -47,6 +49,71 @@ def append_pod_ip_to_env(env):
     return env
 
 
+class ClusterSpec(object):
+    def __init__(self, cluster_spec_json="", cluster_spec=""):
```

Comment:
Since `cluster_spec_json` contains all the information, it seems that there is no need to load `cluster_spec.py`.

The `with_service` and `with_pod` method could be defined in ElasticDL repo.

Reply:
This is for legacy support for cluster_spec.py support. 
Also, users may define a model in the repo and submit it to different clusters. No need to modify the repo if using command arguments or external files.

Reply:
We may remove cluster_spec.py later if all are switched to command arguments.